Cargo.lock

```diff
@@ -115,9 +115,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.58"
```

Comment:
Any particular reason for version change.

Reply:
I did `cargo add anyhow`. Didn't notice change. Should this be reverted ?